src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java

```diff
@@ -14,20 +14,22 @@
 
 package com.google.graphgeckos.dashboard.storage;
 
+import com.google.cloud.datastore.Datastore;
 import com.google.cloud.datastore.DatastoreException;
 import com.google.cloud.datastore.Entity;
 import com.google.cloud.datastore.Query;
 import com.google.cloud.datastore.StructuredQuery.OrderBy;
 import com.google.cloud.datastore.StructuredQuery.PropertyFilter;
-import com.google.cloud.datastore.TimestampValue;
 import com.google.cloud.Timestamp;
 import java.util.ArrayList;
 import java.util.Calendar;
 import java.util.Date;
+import java.util.function.Supplier;
```

Comment:
Since <code>function.Supplier</code> is decapitalised, this should come after <code>java.util.List</code>

Reply:
You are right! Thank you!